### PR TITLE
middleware-1.3.27 releasenotes

### PR DESCRIPTION
**Pull request by volllly**

### CODE REVIEW

**Review by Bluette-C-Riviere — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -11,14 +11,20 @@ These release notes will contain a sum-up of all customer-relevant changes that
 If you're viewing this on GitHub, please navigate to https://docs.fiskaltrust.cloud for a more convenient view of this content and open the section of interest to the left. If you have any questions regarding the changes described here, please directly reach out to us via [feedback+releasenotes@fiskaltrust.cloud](mailto:feedback+releasenotes@fiskaltrust.cloud).
 
 
+## December 20, 2021
+
+### Middleware 1.3.27 (Germany)
+In this version of the Middleware, we updated the FCC used in the `SwissbitCloud` and `DeutscheFiskal` SCUs to protect gainst Log4Shell exploits.
```
Comment (mijomilicevic) [inline README.md:17]: should it be 'against' ?

Comment (mijomilicevic) [inline README.md:17]:
```suggestion
In this version of the Middleware, we updated the FCC used in the `SwissbitCloud` and `DeutscheFiskal` SCUs to protect against Log4Shell exploits.
```

**Review by fiscoal — APPROVED** [review]
Maybe linkifying [Log4Shell](https://en.wikipedia.org/wiki/Log4Shell) for convenience?

After all, the logo took quite some design talent 🤪

image

**Comment by volllly** [comment]
@fiscoal Good idea 🙏 

But I think this is the latest version of the logo 😅

log4shell-keeps-coming
https://www.lunasec.io/docs/blog/log4j-zero-day-severity-of-cve-2021-45046-increased/

**Pull request merged**